include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.h

```diff
@@ -80,6 +80,14 @@ class TimeIntBDF : public TimeIntBDFBase<Number>
   get_velocities_and_times_np(std::vector<VectorType const *> & velocities,
                               std::vector<double> &             times) const;
 
+  void
+  get_pressures_and_times(std::vector<VectorType const *> & pressures,
+                          std::vector<double> &             times) const;
+
+  void
+  get_pressures_and_times_np(std::vector<VectorType const *> & pressures,
+                             std::vector<double> &             times) const;
+
```

Comment:
It could still make sense to have these new functions in the public interface. However, the same code should be used for velocity and pressure in the internal implementation.

Reply:
Yes, I would like to keep the interface, but I am restructuring the internal code :)

Reply:
Yes, I would like to keep the interface, but I restructured the internal code :)